src/datalad_installer.py

```diff
@@ -1926,6 +1958,56 @@ def download_last_successful_artifact(
         else:
             raise RuntimeError("No workflow runs with artifacts found!")
 
+    def get_latest_release_asset(self, repo: str, ext: str) -> dict:
```

Comment:
Please add a unit test or full installation test for this mode

Reply:
We can't really do that, as git-annex would be installed globally and would clash with the git-annex installed by the neurodebian test.

Reply:
Ok, no integration test, but functions to get the artifact should get unit tested

Reply:
Should the tests just test that *something* with the correct extension gets downloaded?  What exactly are you envisioning the tests doing?

Reply:
Yes, mostly a smoke test that something of appropriate size gets downloaded

Reply:
I can check for a specific size if downloading a specific release, but if downloading the latest release (which changes over time), what constitutes "appropriate size"?

Reply:
@yarikoptic Ping; please respond to previous comment.

Reply:
make it greater than 1MB, I don't expect it to go lower (some error html page would likely be less ;) )

Reply:
@yarikoptic Tests added.